File: Kotlin/app/src/main/java/mx/com/yourlawyer/yourlawyer/model/Case.kt
package mx.com.yourlawyer.yourlawyer.model

data class Case(
    val caseId: String,
    val imageURL: String,
    val title: String,
    val description: String,
    val category: String,
    val postedBy: String,
    val postedDate: String,
    val budget: String,
    val location: String,
    val status: String,
    val details: CaseDetails
)
data class CaseDetails(
    val tags: List<String>,
    val requirements: List<String>,
    val urgency: String
)
data class CasesResponse(
    val cases: List<Case>
)

File: Swift/YourLawyer/YourLawyer/Model/Case.swift
//
//  Case.swift
//  YourLawyer
//
//  Created by Alan Ulises on 08/01/25.
//

import Foundation

struct Case: Decodable {
    let caseId: String
    let imageURL: String
    let title: String
    let description: String
    let category: String
    let postedBy: String
    let postedDate: String
    let budget: String
    let location: String
    let status: String
    let details: CaseDetails
}

struct CaseDetails: Decodable {
    let tags: [String]
    let requirements: [String]
    let urgency: String
}

struct CasesResponse: Decodable {
	let cases: [Case]
}

class CaseManager {
	static let shared = CaseManager()
	
	var selectedCase: Case?
	
	private init() {} // Una sola instancia
}
